toke.c

```diff
@@ -4446,7 +4447,10 @@ S_intuit_more(pTHX_ char *s, char *e)
     if (s[0] != '{' && s[0] != '[')
```

Comment:
I don't understand this line.  Apparently it is implementing the prolog comment

>  * if we're outside a pattern and it's not { or [, then return FALSE

But we haven't tested yet if we're inside a pattern

Reply:
This is just a summary of the function.